src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -8,69 +8,110 @@ import {
-        <Route path="/account" component={AccountSubrouter} />
-        <Route path="/admin" component={AdminV1Subrouter} />
-        <Route path="/admin-ng" component={AdminNGSubrouter} />
-
-        <Route
-          path="/login/:venueId/:customToken"
-          component={LoginWithCustomToken}
-        />
-        {/* @debt The /login route doesn't work since we added non-defaulted props to the Login component */}
-        {/*<Route path="/login" component={Login} />*/}
-
-        <Route path="/v/:venueId" component={VenueLandingPage} />
-        <Route path="/e/:step/:venueId" component={VenueEntrancePage} />
-        <Route path="/in/:venueId/admin" component={VenueAdminPage} />
-        <Route path="/in/:venueId" component={VenuePage} />
-        <Route path="/version" component={VersionPage} />
-
-        <Route
-          path="/venue/*"
-          render={(props) => (
-            <Redirect to={venueLandingUrl(props.match.params[0])} />
-          )}
-        />
-
-        <Route
-          path="/sparkleverse"
-          render={() => {
-            window.location.href = SPARKLEVERSE_HOMEPAGE_URL;
-            return <LoadingPage />;
-          }}
-        />
-
-        <Route
-          path="/"
-          render={() => {
-            window.location.href = DEFAULT_REDIRECT_URL;
-            return <LoadingPage />;
-          }}
-        />
-      </Switch>
+      <Suspense fallback={<LoadingPage />}>
+        <Switch>
+          <Route path="/enter" component={EnterSubrouter} />
+          <Route path="/account" component={AccountSubrouter} />
+          <Route path="/admin" component={AdminSubrouter} />
+          <Route path="/admin-ng" component={AdminSubrouter} />
+
+          <Route
+            path="/login/:venueId/:customToken"
+            component={LoginWithCustomToken}
+          />
+          {/* @debt The /login route doesn't work since we added non-defaulted props to the Login component */}
+          {/*<Route path="/login" component={Login} />*/}
+
+          <Route path="/v/:venueId" component={VenueLandingPage} />
+          <Route path="/e/:step/:venueId" component={VenueEntrancePage} />
+          <Route path="/in/:venueId/admin" component={VenueAdminPage} />
+          <Route path="/in/:venueId" component={VenuePage} />
+          <Route path="/version" component={VersionPage} />
+
+          <Route
+            path="/venue/*"
+            render={(props) => (
+              <Redirect to={venueLandingUrl(props.match.params[0])} />
+            )}
+          />
+
+          <Route
+            path="/sparkleverse"
+            render={() => {
+              window.location.href = SPARKLEVERSE_HOMEPAGE_URL;
+              return <LoadingPage />;
+            }}
+          />
+
+          <Route
+            path="/"
+            render={() => {
+              window.location.href = DEFAULT_REDIRECT_URL;
+              return <LoadingPage />;
+            }}
```

Comment:
The only thing changed is the Suspense wrapper, correct?

Reply:
You should get into the habit of using the 'show/hide whitespace changes' button on GitHub's files tab, it will make review WAY easier for you.

![image](https://user-images.githubusercontent.com/753891/124716650-aba01800-df47-11eb-8a3f-73147ea728bf.png)

https://github.com/sparkletown/sparkle/pull/1737/files?w=1#diff-af6c68bbe6a66e4fac17e29585b63f1eee98f31ad8d93402c06deefebbdd615e

![image](https://user-images.githubusercontent.com/753891/124716635-a642cd80-df47-11eb-8d24-48ce9d859cdb.png)